info/guide/guide-chapter-3.md

```diff
@@ -396,6 +471,62 @@ if (( x >  100) ||
     y < 100 && !isFoo()) {}
 ```
 
+`String template` also can be split in white space in string text
+
+**Invalid example:**
+```kotlin
+val nameString = "This is very long string template"
+```
+
+**Valid example:**
+```kotlin
+val nameString = "This is very long" +
+        " string template"
+```
+
+Long `Lambda argument` should be split:
+
+**Invalid example:**
+```kotlin
+val variable = a?.filter { it.elementType == true } ?: null
```

Comment:
I'd choose a slightly different example, because `it.elementType == true` shows a code smell: in most cases, it can be simplified as `it.elementType`.

How about

```kotlin
val variable = a?.filter { it.elementType == IDENTIFIER } ?: null
```

Reply:
Done [dd07c31](https://github.com/saveourtool/diktat/pull/1384/commits/dd07c31fed183aa7b78c96db025ce1e7a26bb77f)